backend/conferences/types.py

```diff
@@ -58,12 +64,36 @@ class ConferenceType(DjangoObjectType):
     topics = graphene.NonNull(graphene.List(graphene.NonNull(TopicType)))
     languages = graphene.NonNull(graphene.List(graphene.NonNull(LanguageType)))
     durations = graphene.NonNull(graphene.List(graphene.NonNull(DurationType)))
+    schedule = graphene.NonNull(
+        graphene.List(graphene.NonNull(ModelScheduleItemType)),
+        date=graphene.String(),
```

Comment:
I think it might be better to use `graphene.types.datetime.Date` so we won't need to parse the date in the resolver :)

Reply:
Done